content/post/go-git-v4.md

```diff
@@ -0,0 +1,73 @@
+---
+author: campoy
+date: 2017-12-07
+title: "Announcing the latest go-git!"
+image: /post/go-git-v4/go-git.png
+description: "After a year of intense work, we're happy to announce the latest and best release of go-git ever. go-git v4 includes many new features, making it the most used and feature complete git library written in Go, and in use on production at companies like source{d} and keybase."
+categories: ["git", "announcement", "OSS", "technical"]
+---
+
+After a bit over a year of intense work, we're happy to announce the
+latest and best release of [go-git](https://github.com/src-d/go-git) yet!
+`go-git v4` includes many new features, making it the most used and feature
```

Comment:
go-git name looks broken in desktop in its first blog occurrence.
![image](https://user-images.githubusercontent.com/2437584/33715587-616ec516-db53-11e7-9307-77f727e2cd00.png)
Is it something relevant?

Reply:
this depends on the browser and width of the screen ... I don't think it's a problem

Reply:
It should be like that in most of the desktop (more than 800of width), but ok, fair enough